codegen.yml

```diff
@@ -0,0 +1,23 @@
+schema: schema.graphql
+documents: 'src/**/*.graphql'
+generates:
+  src/__generated__/types.ts:
+    - typescript
+  src/:
+    preset: near-operation-file
+    presetConfig:
+      folder: __generated__
+      extension: .tsx
+      baseTypesPath: __generated__/types.ts
+    plugins:
+      - typescript-operations
+      - typescript-react-apollo
+    config:
+      namingConvention: keep
+      nonOptionalTypename: true
+      dedupeOperationSuffix: true
+      withComponent: true
+      withHooks: true
+      withHOC: false
+      reactApolloVersion: 2
+      # documentMode: documentNode
```

Comment:
**2) `documentMode: documentNode` does not includes DocumentNode from fragments** 
- just uncomment option in `codegen.yaml`
- `yarn watch` 
- open `Orders` tab
- and you will get an error that fragment `OrderRow_order` is unknown from graphql server

Reply:
I see, I think this PR fixes that, right? https://github.com/dotansimha/graphql-code-generator/pull/3171

Reply:
Yep, this PR exactly solves this problem 🎉